status-im/status-react
# Update react-native-swiper

- update react-native-swiper to latest working version (1.5.11 and 1.5.12 are more recent but don't work, see https://github.com/status-im/status-react/issues/1915)
- fixes `Cannot scroll transaction history when app is syncing` and `cannot scroll to the last 2 transactions help wanted wallet`
- remove bottom margin hack in other screens with similar bug

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
fixing #1893 and #1902

### Summary:
[comment]: # (Summarise the problem and how the pull request solves it)
The probable was that the react-native-swiper component was using the height of the full window without taking into account the other components present on the screen such as the tab and it got fixed upstream. This bug already affected other screen in the past and the workaround was to had bottom margin to it which is not necessary anymore

### Steps to test:
- Open Status
- recover an account with transactions
- check that all transactions history tab is scrollable while syncing and all transactions are visible
- check that chat list, discover and contact list are behaving fine on iOS and different Android screen size (especially area shown in the red rectangle when scrolling to the bottom of the screen as shown in screenshot)

![screenshot_1506101801](https://user-images.githubusercontent.com/1181225/30757068-9036a24a-9fcd-11e7-9266-2b48972bdf72.png)

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready
